beacons-data/export_legacy_beacons.sql

```diff
@@ -14,4 +14,5 @@ SELECT
 	UPPER(data -> 'owner' ->> 'county')AS owner_county,
 	UPPER(data -> 'owner' ->> 'townOrCity')AS owner_town_or_city,
 	UPPER(data -> 'owner' ->> 'country')AS owner_country
-FROM legacy_beacon;
\ No newline at end of file
+FROM legacy_beacon
+WHERE last_modified_date >= '2013/01/01';
```

Comment:
What is the reason on this date? If it's a rolling 8 years, might be worth having something like:
```
WHERE last_modified_date >= NOW() - INTERVAL '8 YEARS'
```

I know all modern beacons are newer than 8 years, but would be worth adding this in to the modern script to future proof.

Reply:
Linda requested a filter on the code so it would only return legacy beacons whose Last Modified Date is later than 2013. This is because she wants to contact customers who've updated within the last 8 years: this would point to a requirement for a rolling 8 years I think so I'll make that change now